.github/workflows/release-drafter.yml

```diff
@@ -49,7 +49,32 @@ jobs:
           ./gradlew publishMavenJavaPublicationToMavenRepository -Dbuild.snapshot=false -Dbuild.version=0.${{ steps.get_data.outputs.version }} && tar -C build -cvf traffic-capture-artifacts.tar.gz repository
       - name: Build Docker Images
         run: |
-          ./deployment/cdk/opensearch-service-migration/buildDockerImages.sh
+          ./deployment/cdk/opensearch-service-migration/buildDockerImages.sh -Dbuild.snapshot=false -Dbuild.version=0.${{ steps.get_data.outputs.version }}
+      - name: Generate SBOM for migration_console
+        uses: anchore/sbom-action@v0
+        with:
+          image: migrations/migration_console:latest
+          artifact-name: opensearch-migrations-console-sbom.spdx.json
+      - name: Generate SBOM for traffic_replayer
+        uses: anchore/sbom-action@v0
+        with:
+          image: migrations/traffic_replayer:latest
+          artifact-name: opensearch-migrations-traffic-replayer-sbom.spdx.json
+      - name: Generate SBOM for capture_proxy
+        uses: anchore/sbom-action@v0
+        with:
+          image: migrations/capture_proxy:latest
+          artifact-name: opensearch-migrations-traffic-capture-proxy-sbom.spdx.json
+      - name: Generate SBOM for reindex_from_snapshot
+        uses: anchore/sbom-action@v0
+        with:
+          image: migrations/reindex_from_snapshot:latest
+          artifact-name: opensearch-migrations-reindex-from-snapshot-sbom.spdx.json
+      - name: Generate SBOM for artifacts
+        uses: anchore/sbom-action@v0
```

Comment:
How did you come to pick this GitHub action, looks like it hasn't been fully released yet?

Reply:
Lets use `v0.X.Y` to lock to a specific rather than just `v0` - dependabot can manage these updates.

Reply:
Updated